src/Benchmarks.ServerJob/Scenario.cs

```diff
@@ -13,6 +13,7 @@ public enum Scenario
     {
         Plaintext,
         Json,
+        Copy,
```

Comment:
I'll rename the scenario to `CopyToAsync`, but will keep the script name `post.lua` - all the script is going is posting data. From the script's perspective there's nothing about `CopyToAsync` going on.